lisa/target.py

```diff
@@ -197,13 +198,14 @@ class used under the hood, and avoids messing up with ``devlib``
 
     CONF_CLASS = TargetConf
     INIT_KWARGS_KEY_MAP = {
+        'kernel_src': ['kernel_src'],
         'devlib_excluded_modules': ['devlib', 'excluded-modules']
     }
 
     def __init__(self, kind, name='<noname>', tools=[], res_dir=None,
         plat_info=None, workdir=None, device=None, host=None, port=None,
-        username='root', password=None, keyfile=None, devlib_platform=None,
-        devlib_excluded_modules=[]
+        username='root', password=None, keyfile=None, kernel_src=None,
+        devlib_platform=None, devlib_excluded_modules=[]
```

Comment:
Why not just add it at the end?

Reply:
To preserve the hierarchy, so it's at the end of the top level